src/js/modules/articles/components/ArticleFeaturedMedia.js

```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import injectSheet from "react-jss";
 
+import CreditLine from "./CreditLine";
```

Comment:
Removed because we are not using it anywhere else, it is a small component, and it doesn't require any processing like `Byline` does.